src/engine/engine_common.c

```diff
@@ -178,17 +179,12 @@ static void ocf_engine_update_req_info(struct ocf_cache *cache,
 			entry->re_part = true;
 			req->info.re_part_no++;
 		}
+	}
 
-		break;
-	case LOOKUP_INSERTED:
-	case LOOKUP_REMAPPED:
+
+	if (entry->status == LOOKUP_INSERTED ||
+			entry->status == LOOKUP_REMAPPED) {
 		req->info.insert_no++;
-		break;
-	case LOOKUP_MISS:
-		break;
-	default:
-		ENV_BUG();
```

Comment:
Isn't it needed in case of entry->status is neither of those lookups?

Reply:
Not sure. I've restored a BUG_ON